src/main/resources/db/changelog/changes/08-create-category-table-and-change-book-table.yaml

```diff
@@ -0,0 +1,59 @@
+databaseChangeLog:

```

Comment:
rename file, you don't change book table
it's better create separate changelogs for create category and create book-category